eng/tools/spec-gen-sdk-runner/src/types.ts

```diff
@@ -19,6 +19,15 @@ export interface SpecGenSdkCmdInput {
   headBranch?: string;
 }
 
+/**
+ * Data for the API view request.
+ */
+export interface APIViewRequestData {
+  packageName: string;
+  filePath: string;
+  language: string;
```

Comment:
Language will always be the same in a pipeline run. Can you use `SpecGenSdkArtifactInfo.language`? In addition, language here is like this format: 'azure-sdk-for-python'. I think you need to extract the 'python' from this value when using it in API view script.

Reply:
The correct language is in genertedOutput maybe I can just use that.

Reply:
It's not necessary to access `generatedOutput`. Instead, extract the short language name from the string of `SpecGenSdkArtifactInfo.language`.